service/ziti-tunnel/service/orphaned-identities.go

```diff
@@ -0,0 +1,95 @@
+package service
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/openziti/desktop-edge-win/service/ziti-tunnel/config"
+)
+
+func ScanAndCopyFromBackup() error {
+	srcBackUpPaths := [2]string{"\\Windows.~BT", "\\Windows.old"}

```

Comment:
the .join will add the \ for you and can be removed here. also go has an amazing language feature of string literals using the ` delimiter.
```
a := `\b\c`
```